docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/pdq/PDQ.js

```diff
@@ -19,8 +19,11 @@ document.addEventListener('DOMContentLoaded',onDOMContentLoaded);
 /* TODO: create a field in the content type for 'hp-patient-toggle-link' */
 const moveToggle = () => {
   const toggle = document.querySelector('.pdq-hp-patient-toggle', '#cgvBody');
```

Comment:
As far as I know, the api for document.querySelector does not handle more than one argument. I see it being used with multiple params in a couple places here. I believe the 2+ arguments are simply being ignored.

Reply:
You're correct. This should be one string argument with CSS selectors separated by a comma `document.querySelector('.pdq-hp-patient-toggle, #cgvBody')`

Reply:
Correction, that second argument is intended to provide context to the selector the way jQuery does it. It should be `document.querySelector('#cgvBody .pdq-hp-patient-toggle')`